src/pages/EditProfilePage/EditProfilePage.jsx

```diff
@@ -1,21 +1,40 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { useNavigate } from "react-router-dom";
-import ProfilePicture2 from "../../components/ProfilePicture2/ProfilePicture2"; 
+import ProfilePicture2 from "../../components/ProfilePicture2/ProfilePicture2";
 import Button from "../../components/shared/Button/Button";
 import "./EditProfilePage.css";
 import GoBackButton from "../../components/shared/GoBackButton/GoBackButton";
+import Parse from "parse/dist/parse.min";
 
 function EditProfilePage() {
   const navigate = useNavigate();
-  const [editMode, setEditMode] = useState(false); //I moved this from profilepage
-  const [name, setName] = useState("Alice"); //I moved this from profilepage
+  const [editMode, setEditMode] = useState(false);
+  const [name, setName] = useState("");
+
+  useEffect(() => {
+    const currentUser = Parse.User.current();
+    if (currentUser) {
+      setName(currentUser.get("name") || "Alice"); // fallback for al least something
```

Comment:
Instead of "Alice" I would prefer to say "Anonymous" or something like that

Reply:
Sure. Will change it to anonymous.